### PR TITLE
Modifies schaake_partitioning.hpp code variables from upper to lower …

### PR DESCRIPTION
**Pull request by stcui007**
Add unit test for Pdm03.h and modify Hymod related codes
Pdm03_Test.cpp
Modified Hymod.h
Modified HymodTest.cpp

Modified the code to be consistent with the style of T-shirt model.
New version tested and compared with previous versions of the code.

## Additions
Pdm03_Test.cpp

## Removals

-

## Changes
Hymod.h
HymodTest.cpp

## Testing
Unit test for Pdm03.h using Pdm03_Test.cpp passed

## Notes
Could not run both Pdm03_Test.cpp and HymodTest.cpp simultaneously, as they both test Pdm03.h. Multiple declaration error.

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

## Testing checklist (automated report can be put here)

1.

### Target Environment support

- [ ] Linux

### CODE REVIEW

**Comment by robertbartel** [comment]
@stcui007, could you rebase your branch used for this PR?  In reviewing it, I tried to run the unit tests, but I get compilation errors when building the _test_unit_ target.  At first glance they look like things that have already been resolved in the main repo, but I think it is best to be safe and make sure tests pass before approving.

**Comment by stcui007** [comment]
I am planning to do a unit test this afternoon. If everything works ok,
I'll upload the new codes. The current test code is probably not conforming
to the standard.

Shengting

File: CMakeCache.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# This is the CMakeCache file.
```
Comment (hellkite500) [inline CMakeCache.txt:1]: CMakeCache should not be indexed.

File: CMakeFiles/cmake.check_cache [hunk 2]
```diff
@@ -0,0 +1 @@
+# This file is generated by cmake for dependency checking of the CMakeCache.txt file
```
Comment (hellkite500) [inline CMakeFiles/cmake.check_cache:1]: Generated files/directories should not be commited/indexed.

File: test/geojson/Feature_Test.cpp [hunk 3]
```diff
@@ -302,7 +302,7 @@ TEST_F(Feature_Test, ptree_test) {
       "} "
```
Comment (hellkite500) [inline test/geojson/Feature_Test.cpp:302]: Changes to geojson might conflict here.

**Comment by hellkite500** [comment]
Need to sort out with #59 and #63.  This PR is messy with a lot of duplicate changes from other commits it looks like.  Will work to sort out.

**Comment by hellkite500** [comment]
Replaced by #102

**Pull request closed without merging**